tests/config/tikv.toml

```diff
@@ -25,3 +26,10 @@ hibernate-regions-compatible=false
 ca-path = "/tmp/backup_restore_test/certs/ca.pem"
 cert-path = "/tmp/backup_restore_test/certs/tikv.pem"
 key-path = "/tmp/backup_restore_test/certs/tikv.key"
+
+[security.encryption]
+data-encryption-method = "aes256-ctr"
+
+[security.encryption.master-key]
+type = "file"
+path = "/tmp/file"
```

Comment:
please use a more specific file name 😓.
preferably put it inside `/tmp/backup_restore_test/`.